apps/testingaccessibility/src/pages/learn/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import {GetServerSideProps} from 'next'
+import {stripe} from 'utils/stripe'
+import {Stripe} from 'stripe'
+import {getDecodedToken} from '../../utils/get-decoded-token'
+import {session} from 'next-auth/core/routes'
+import {getSdk} from '../../lib/prisma-api'
+import {Purchase} from '@prisma/client'
+
+export const getServerSideProps: GetServerSideProps = async ({req, query}) => {
+  const sessionToken = await getDecodedToken(req)
+  const {getPurchasesForUser} = getSdk()
+
+  if (sessionToken && sessionToken.sub) {
+    const purchases = await getPurchasesForUser(sessionToken.sub)
```

Comment:
since we can validate their purchases here on the server we can load the actual content from wherever we need to